src/medCore/database/medDatabaseRemover.cpp

```diff
@@ -224,7 +225,12 @@ void medDatabaseRemover::removeSeries ( int patientDbId, int studyDbId, int seri
     if ( query.next() )
     {
         thumbnail = query.value ( 0 ).toString();
-        this->removeFile ( thumbnail );
+
+        medAbstractDbController * dbc = medDataManager::instance()->controllerForDataSource(d->index.dataSourceId());
+        if (thumbnail == dbc->metaData(d->index,  medMetaDataKeys::ThumbnailPath.key()))
```

Comment:
These lines allow to check the updated location of the thumbnail with the current location of the patient/study/serie which is removing. For instance, if you move a data to an other patient, and if you remove the first patient, your thumbnail of the moved data won't be removed.

... It's a bit complicated, said like that ^^'